### DISCUSSION TITLE
The icons are not aligned

### DISCUSSION THREAD

**Discussion by zhanxiaox** [community/community#111284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/16477289/647990d4-c577-477b-b2f5-6c11633d28c7)
like this.

**Comment by thilinaabey** [comment]
Its the normal preview of GitHub

**Comment by SebastienXia** [comment]
You're not the only one to have noticed this but don't worry, it's normal.
